test/controllers/home_controller_test.rb

```diff
@@ -5,76 +5,83 @@ class HomeControllerTest < ActionDispatch::IntegrationTest
     get '/'
 
     assert_response :success
-    assert_includes response.parsed_body, "Home#index"
+    assert_includes response.parsed_body, "| Home</title>"
```

Comment:
i think it's best to have the page specific title first, then more generic like opeNode:

<title>Home | opeNode</title>